src/uwtools/rocoto.py

```diff
@@ -174,21 +174,82 @@ def _add_task(self, e: Element, config: dict, taskname: str) -> None:
         if STR.dependency in config:
             self._add_task_dependency(e, config[STR.dependency])
 
+    @property
+    def _dependency_constants(self):
+        """
+        Returns a tuple for each of the resepctive dependency types.
+        """
+        operands = (STR.datadep, STR.taskdep, STR.timedep)
+        operators = (STR.and_, STR.nand, STR.nor, STR.not_, STR.or_, STR.xor)
+        strequality = (STR.streq, STR.strneq)
+
+        return operands, operators, strequality
+
```

Comment:
I added this `@property` method to address DRY out comment. I passed the return value from the call in `_add_task_dependency` to `_add_task_dependency_operator` to reuse this information.